crates/ide-completion/src/context/analysis.rs

```diff
@@ -943,6 +943,7 @@ impl<'a> CompletionContext<'a> {
                         }
                     }
                 },
+                ast::RecordExpr(it) => make_path_kind_expr(it.into()),
```

Comment:
This line also needs to go into the match in the `// completing inside a qualifier` annotated part as well